### ISSUE TITLE
New seq2seq interface(basic_decoder) does not support sampled softmax?

### ISSUE THREAD

**Issue by chenghuige** [tensorflow/tensorflow#9833]
basic_decoder init function has param output_layer which must be a type of layer like Dense.
But for using sampled softmax we need some code like below, as w_t and v is needed by tf.nn.sampled_softmax_loss 
      with tf.variable_scope('output_projection'):
          self.w_t = melt.variable.get_weights_truncated('w',   
                                               [vocab_size, num_units],   
                                               stddev=FLAGS.weight_stddev)   
          #weights  
          self.w = tf.transpose(self.w_t)    
          #biases  
          self.v = melt.variable.get_weights_truncated('v',     
                                              [vocab_size], 
                                              stddev=FLAGS.weight_stddev)   
for old seq2seq interface, we can just pass output_function like 
      def output_fn(output):  
          return tf.nn.xw_plus_b(output, self.w, self.v )  

How can we do sampled softmax with seq2seq new internface ?

**Comment by girving** [comment]
This question is better asked on  [StackOverflow](http://stackoverflow.com/questions/tagged/tensorflow) since it is not a  bug or feature request. There is also a larger community that reads questions there. Thanks!
